src/igvfd/searches/configs/workflow.py

```diff
@@ -7,7 +7,7 @@
 def workflow():
     return {
         'facets': {
-            'analysis_steps.analysis_step_versions.software_versions.software.name': {
+            'analysis_steps.analysis_step_versions.software_versions.software.title': {
```

Comment:
For consistency I changed this to title as well. Don't think name makes sense vs. title